src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -35,12 +37,21 @@
 RESULTS_CHANNEL = "d3a-results"
 ZIP_RESULTS_CHANNEL = "d3a-zip-results"
 ZIP_RESULTS_KEY = "d3a-zip-results-key/"
+HEARTBEAT_CHANNEL = "d3a-heartbeat"
+PULSE_RATE = 5  # in secs
 
 
 def utf8len(s):
     return len(s.encode('utf-8')) / 1000.0
 
 
+class RepeatingTimer(Timer):
```

Comment:
I would suggest to move the RepeatTimer class from d3a-web to d3a-interface, and reuse it in both places. That way we will not have 2 clones of the same class across our repos